packages/react/src/primitives/Alert/Alert.tsx

```diff
@@ -37,7 +37,11 @@ const AlertPrimitive: Primitive<AlertProps, typeof Flex> = (
   return (
     !dismissed && (
       <Flex
-        className={classNames(ComponentClassNames.Alert, className)}
+        className={classNames(
+          ComponentClassNames.Alert,
+          className,
+          variation ? `${ComponentClassNames.Alert}--${variation}` : null
```

Comment:
What do you think of having a util function for creating these BEM modifier classes?

```ts
const classNameModifier = ({base, modifier}) => {
  return modifier ? `${base}--${modifier}` : null
}
```

Could use like:

```tsx
className={classNames(
          ComponentClassNames.Alert,
          className,
          classNameModifier({base: ComponentClassNames.Alert, modifier: variation})

```